src/app/app-routing.module.ts

```diff
@@ -1,7 +1,10 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { RegisterComponent } from './auth/register/register.component';
 
-const routes: Routes = [];
+const routes: Routes = [
+  {path:'',redirectTo:'auth',pathMatch: 'full'},
```

Comment:
add space around `{` and `}`
to be
`{ path:'',redirectTo:'auth',pathMatch: 'full' },`